### DISCUSSION TITLE
Tiff to png conversion results in big file size

### DISCUSSION THREAD

**Discussion by vlad8090** [ImageMagick/ImageMagick#5226] [Help] [answered]
This was initially found with Magick.NET, but I can recreate this with ImageMagick with command:

convert original.tif -background none -layers merge -resize 750 result.png

Results in a file with size 2.6 mb. Photoshop resaves it to way smaller size – 77 kb.
Recreated with latest version.

Is there a way to reduce file size?
I tried few compressions – did not help.

The file: https://drive.google.com/file/d/1B3ZYWueZbJWmeBqaUw_LPsDoJItsG1Pd/view?usp=sharing

Thanks

**Comment by snibgo** [comment, marked as answer]
Using ...
```
magick result.png -verbose info:
```
... shows that the file contains a profile "tiff:37724" that is 4 MB. I don't know what that is. We can remove just that:
```
magick original.tif -background none -layers merge -resize 750 +profile "tiff:37724" x.png
```
The result is 81283 bytes. We could convert it to sRGB and remove the small ICC profile, and probably also remove the 8bim and xmp profiles.

**Reply by vlad8090** [reply]
Thank  you. Removed this profile from png. 
Magick.net code:
image.RemoveProfile("tiff:37724");
